src/main/java/nl/tudelft/cse1110/andy/execution/metatest/library/LibraryMetaTest.java

```diff
@@ -99,6 +99,10 @@ private ResultBuilder runMetaTest(Context ctx, DirectoryConfiguration dirCfg, Fi
         metaCtx.setDirectoryConfiguration(metaDirCfg);
         metaCtx.setLibrariesToBeIncluded(ctx.getLibrariesToBeIncluded());
 
+        if(!ctx.isSecurityEnabled()){
```

Comment:
Are these meta tests run when running Andy through Maven?

Reply:
Yes, they are. Running it through Maven runs the same steps as running it normally.

 Meta tests are run in their own "sub-context" and we want to configure this context not to use security if the main context doesn't.

Reply:
LGTM then!